### PR TITLE
Feature/jenkins updates

### PR DESCRIPTION
**Pull request by cookersjs**
Minor changes to the GO pipeline

- DB dump files are now timestamped
- Update to environmental variables
- Changed person.id variable in the actual code to be personId, which is how other modules in the pipeline call it

### CODE REVIEW

**Review by SolomonShorser-OICR — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -73,9 +75,10 @@ pipeline {
 		stage('Post: Backup gk_central after modifications'){
 			steps{
 				script{
+				    def timestamp = new Date().format("yyyy-MM-dd-HHmmss")
```
Comment (jweiser) [inline Jenkinsfile]: I noticed this same implementation for timestamp was used in the ChEBI update Jenkins file - could a function be created, put into a util module and imported wherever needed by Jenkins files in the release process (similar to my comment about the release number in the ChEBI update PR - https://github.com/reactome/release-chebi-update/pull/3#discussion_r503067985.

File: Jenkinsfile [hunk 2]
```diff
@@ -73,9 +75,10 @@ pipeline {
...
 					withCredentials([usernamePassword(credentialsId: 'mySQLCuratorUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-						def central_after_update_go_update_dump = "${env.GK_CENTRAL}_${currentRelease}_after_go_update.dump"
-						sh "mysqldump -u$user -p$pass -h${env.CURATOR_SERVER} ${env.GK_CENTRAL} > $central_after_update_go_update_dump"
+						def central_after_update_go_update_dump = "${env.GK_CENTRAL_DB}_${env.RELEASE_VERSION}_after_go_update_${timestamp}.dump"
```
Comment (jweiser) [inline Jenkinsfile]: Similar to https://github.com/reactome/release-chebi-update/pull/3#discussion_r503068786

**Review by jweiser — DISMISSED** [review]
Looks good.  No blockers, but similar suggestions to the ChEBI update Jenkins file PR

**Review by SolomonShorser-OICR — COMMENTED** [review]
This PR depends on the other one for the shared library. Maybe this PR should be put into a Draft state until the one it depends on is complete & merged?

**Comment by cookersjs** [comment]
You can actually specify the branch to pull the shared library from outside of the Jenkinsfiles -- right now it is specified to pull from url:release-jenkins-utils, branch: feature/shared-groovy-library. Once it is merged, it is just one configuration that needs to be updated

**Comment by SolomonShorser-OICR** [comment]
I know that Jenkins can point to whatever branch you tell it to, I was just trying to point out that opening a PR for changes that depend on code in another unmerged PR is a tricky thing... what happens if _this_ PR gets approved before the one it depends on, and then changes in that other PR trigger a need for changes in this one?

**Comment by cookersjs** [comment]
> I know that Jenkins can point to whatever branch you tell it to, I was just trying to point out that opening a PR for changes that depend on code in another unmerged PR is a tricky thing... what happens if _this_ PR gets approved before the one it depends on, and then changes in that other PR trigger a need for changes in this one?

Fair enough -- then I think my request will be to at least have these tentatively approved (after review of most recent changes) and then that way if nothing ends up changing I can just merge them instead of having 15+ PRs in need of re-review

File: Jenkinsfile [hunk 3]
```diff
@@ -104,25 +87,14 @@ pipeline {
...
-					sh "mv go-update-v${env.RELEASE_VERSION}-reports.tgz data/"
-					sh "aws s3 --no-progress --recursive cp databases/ $s3Path/databases/"
-					sh "aws s3 --no-progress --recursive cp logs/ $s3Path/logs/"
-					sh "aws s3 --no-progress --recursive cp data/ $s3Path/data/"
-					sh "rm -r databases logs data reports"
+					def releaseVersion = utils.getReleaseVersion()
+					def dataFiles = ["src/main/resources/go.obo", "src/main/resources/ec2go", "go-update-v${releaseVersion}-reports.tgz"]
+					// GO Update log files are already in a folder called 'logs', so an empty list is passed.
+					def logFiles = []
+					utils.cleanUpAndArchiveBuildFiles("go_update", dataFiles, logFiles)
```
Comment (jweiser) [inline Jenkinsfile]: Just wondering, has the change to archiving been tested?

Comment (cookersjs) [reply Jenkinsfile]: Archiving in this manner was used during the v75 release

**Review by SolomonShorser-OICR — APPROVED** [review]

**Review by SolomonShorser-OICR — APPROVED** [review]

File: Jenkinsfile (hunk 3 above)
Comment (cookersjs) [reply Jenkinsfile]: Yes, it was used during v75

**Pull request merged**